recipes/ligatehaplotypes/meta.yaml

```diff
@@ -0,0 +1,27 @@
+package:
+  name: ligatehaplotypes
+  version: '1.0'
+
+source:
+  fn: ligateHAPLOTYPES.tgz
+  url: https://mathgen.stats.ox.ac.uk/genetics_software/shapeit/files/ligateHAPLOTYPES.tgz
+  md5: cd0fe75c2008999ef4c7772c075b11f0
+
+build:
+  number: 0
+  skip: True  # [osx]
+
+requirements:
+  build:
+    - gcc
+  run:
+    - gcc
```

Comment:
libgcc, please look at our documentation for build c/c++ package for more information's.